knative-operator/deploy/resources/dashboards/grafana-dash-knative-eventing-broker.yaml

```diff
@@ -215,7 +215,7 @@ data:
           "thresholds": "",
           "timeFrom": null,
           "timeShift": null,
-          "title": "Broker (broker-ingress): Success Rate  (2xx Event, rate per minute)",
+          "title": "Broker (broker-ingress): Success Rate  (2xx Event, fraction rate, over 1m window)",
```

Comment:
@skonto Note the double spacing between "Success Rate" and "(2xx..." - perhaps it wasn't intentional.